src/ed_auth/application/features/auth/dtos/create_user_verify_dto.py

```
@@ -1,7 +1,8 @@
-from typing import TypedDict
 from uuid import UUID
 
+from pydantic import BaseModel
```

Pylint (3.3.7)

[error] Unable to import 'pydantic'

(E0401)